priv/repo/structure.sql

```diff
@@ -1524,7 +1524,7 @@ ALTER SEQUENCE user_tasks_id_seq OWNED BY user_tasks.id;
 --
 
 CREATE TABLE users (
-    id bigint NOT NULL,
+    id integer NOT NULL,
```

Comment:
You should probably do a `mix ecto.reset`. This is reverting the ecto change pushed through one of my recent PRs.

Reply:
I have done one a number of times. This change seems to be happening at random.